evaluation.py

```diff
@@ -237,7 +237,97 @@ def predict_keypoints(self,generator, location):
                 image_ids.append(metadata['src_set_image_id'])
                 j+=1
         self.write_to_json_file(list_of_predictions, location)
-        return image_ids
+        return image_ids, list_of_predictions
+
+    def pck_eval(self, generator, location, annotation_file):
```

Comment:
is `location` a save location or read location?

Reply:
It's used to save the location of the predicted keypoints json in `predict_keypoints`, I used the function Rafay had already made to return the predicted keypoints as well as saving them to the json for OKS.

Reply:
Small nit: could you add `save` or something similar to indicate it's for writing?